src/Data/Characters/Chongyun/index.tsx

```diff
@@ -238,7 +238,8 @@ const sheet: ICharacterSheet = {
       },
       passive1: talentTemplate("passive1", tr, passive1),
       passive2: talentTemplate("passive2", tr, passive2, [{
-        node: infoMut(dmgFormulas.passive2.dmg, { key: `char_${key}:passive2` })
+        canShow: data => data.get(input.asc).value >= 4,
```

Comment:
this `canShow` is redundant. the addition of `greaterEq(Input.asc,4,...)` in line 95 should disable/enable the display of this node properly.

Reply:
I'll remove it later, after i merge this in.